test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1421,21 +1431,24 @@ public void help() throws Exception {
     }
   }
 
+  // TODO - evaluate this test is testing what is expected or history is working.
   @Test
   public void history() throws Exception {
     final String table = name.getMethodName();
 
     ts.exec("history -c", true);
     ts.exec("createtable " + table);
     ts.exec("deletetable -f " + table);
+    // TODO - this may be passing only because method name is history.
     ts.exec("history", true, table, true);
+    // TODO - what is this testing?
```

Comment:
This is ideally testing to see if the history command ran previously shows up in this history check. Which it does in the shell. Unsure if this is working correctly due to the issue mentioned above.

Reply:
What I was marking - testing for a table named "history" and a string "history" seems either redundant - or was trying to test something else.  It could be that the string test was to see that history was returned as part of the command and when something else besides history is the table name then it makes more sense.

Reply:
I believe it was just to test that 'history' showed up in the history. I do agree that it didn't make sense since the table was also called 'history'.